src/test/javascript/CommonJsTest.js

```diff
@@ -78,6 +78,71 @@ QUnit.test('sortDate(x, y)', function(assert) {
         '25 April 1999 2:00PM - 25 April 1999 2:01PM');
 });
 
+QUnit.test('getPointValue(s, ditchZero)', function(assert) {
```

Comment:
That is also my question.
There is a code segment in `getPointValue()`
``` javascript
    if (s0.lastIndexOf('<') !== -1) {
        s0 = s0.substring(0, s0.lastIndexOf('<'));
        s0 = s0.substring(s0.lastIndexOf('>') + 1);
    }
```
which seems like to get the text in `<html_tag>TEXT<html_tag>`, but seems that it never used. I even comment it out and pass the test cases. I am hesitating to remove it.